test/parking_location/configs/parking_location_choice.csv

```diff
@@ -0,0 +1,24 @@
+Description,Expression,no_segmentation
+Person is worker,_person_is_worker@((df.person_type == 1) | (df.person_type == 3)),1
```

Comment:
person type 3 is university student in ActivitySim.  Do you mean part time worker with person_type == 2?  Suggest just using the "is_worker" variable.  Also, the person type variable in ActivitySim is named ptype and not person_type.